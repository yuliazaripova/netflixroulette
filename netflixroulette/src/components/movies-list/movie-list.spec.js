import * as React from 'react';

import MoviesList from './movies-list';

describe("MoviesList component", () => {
  it("should render MoviesList component", () => {
    const testProps = [{
        id: 1,
        title: "test title",
        vote_average: 1,
        release_date:  "2017-12-13",
        poster_path: "test poster",
        overview: "test overview",
        runtime: 1,
        genres: ["test genre1", "test genre2"],
    }, 
    {
      id: 2,
      title: "test title2",
      vote_average: 2,
      release_date:  "2017-12-13",
      poster_path: "test poster2",
      overview: "test overview2",
      runtime: 1,
      genres: ["test genre1", "test genre2"],
  }]
    const component = shallow(<MoviesList data={testProps}/>);
    expect(component).toMatchSnapshot();
  })
})