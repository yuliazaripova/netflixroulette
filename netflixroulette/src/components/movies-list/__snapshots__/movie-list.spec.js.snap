// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList component should render MoviesList component 1`] = `
<div
  className="movie-list"
>
  <Movie
    data={
      Object {
        "genres": Array [
          "test genre1",
          "test genre2",
        ],
        "id": 1,
        "overview": "test overview",
        "poster_path": "test poster",
        "release_date": "2017-12-13",
        "runtime": 1,
        "title": "test title",
        "vote_average": 1,
      }
    }
    key="1"
  />
  <Movie
    data={
      Object {
        "genres": Array [
          "test genre1",
          "test genre2",
        ],
        "id": 2,
        "overview": "test overview2",
        "poster_path": "test poster2",
        "release_date": "2017-12-13",
        "runtime": 1,
        "title": "test title2",
        "vote_average": 2,
      }
    }
    key="2"
  />
</div>
`;
